appveyor.yml

```diff
@@ -1,5 +1,9 @@
 ---
 
+# TODO(binji): roll back to previous image because MSYS2 6.3.0 seems to be
+# broken.
+os: Previous Visual Studio 2015
```

Comment:
This rolls back the entire image, got the magic incantation here: https://www.appveyor.com/updates/